packs/feats/skilled-climber.json

```diff
@@ -31,8 +31,25 @@
         "rules": [
             {
                 "key": "BaseSpeed",
-                "type": "climb",
+                "selector": "climb",
                 "value": 10
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "climbing-cavern-wall-or-tree"
+                ],
+                "selector": "climb",
+                "value": "@actor.attributes.speed.total"
+            },
+            {
+                "key": "RollOption",
+                "label": "Climbing Cavern Wall Or Tree",
```

Comment:
Please localize this in `re-en.json`, likely under something like `PF2E.SpecificRule.Kashrishi.SkilledClimber.ClimbToggle`

We try to keep plain text in rule elements as localization strings only.